samples/client/http.go

```diff
@@ -0,0 +1,66 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+func shortHandsWithContext(context.Context) {
+	resp, err := http.Get("http://localhost:8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+	resp.Body.Close()
+
+	resp, err = http.Head("http://localhost:8080")
+	if err != nil {
+		log.Fatal(err)
+	}
+	resp.Body.Close()
+
+	resp, err = http.Post("http://localhost:8080", "text/plain", strings.NewReader("Body"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	resp.Body.Close()
+
+	resp, err = http.PostForm("http://localhost:8080", url.Values{"key": {"value"}})
+	if err != nil {
+		log.Fatal(err)
+	}
+	resp.Body.Close()
+}
+
+func shortHandsWithRequest(_ *http.Request /* for context */) {
```

Comment:
Wait, orchestrion is capable is renaming it and using the generated name in generated and make sure the name is not in conflict with anything ? That's wild.

Reply:
So will the like be considered generated ? Or do you put only the variable name on a line alone?

Reply:
The variable name will be in generated context; but this won't ever show up in a stack trace anyway...

But yes, it generates a name that is most likely not to cause a conflict (due to the abundance of `_`s in it)... Obviously we can be even smarter in the future, but the complexity of that didn't seem worth the effort at this point...